server/match_handler.go

```diff
@@ -106,11 +106,12 @@ type MatchHandler struct {
 
 	// Match state.
 	state interface{}
+
+	TickRate int
```

Comment:
Just noticed we already have this field above named `Rate`. Can we simplify to just reuse that one?